src/test/java/com/bobocode/blyznytsia/bibernate/util/EntityUtilTest.java

```diff
@@ -0,0 +1,62 @@
+package com.bobocode.blyznytsia.bibernate.util;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.bobocode.blyznytsia.bibernate.exception.MalformedEntityException;
+import com.bobocode.blyznytsia.bibernate.testdata.entity.AnnotatedSampleEntity;
+import com.bobocode.blyznytsia.bibernate.testdata.entity.EntityWIthMultipleNonIdFields;
+import com.bobocode.blyznytsia.bibernate.testdata.entity.EntityWithMultipleIds;
+import com.bobocode.blyznytsia.bibernate.testdata.entity.EntityWithoutId;
+import com.bobocode.blyznytsia.bibernate.testdata.entity.SampleEntity;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+class EntityUtilTest {
+
+  @Test
+  void resolveEntityTableNameWithoutAnnotation() {
+    assertEquals("sample_entity", EntityUtil.resolveEntityTableName(SampleEntity.class));
+  }
+
+  @Test
+  void resolveEntityTableNameWithAnnotation() {
+    assertEquals("custom_entity_table_name", EntityUtil.resolveEntityTableName(AnnotatedSampleEntity.class));
+  }
+
+  @Test
+  void resolveFieldColumnValueWithoutAnnotation() throws NoSuchFieldException {
+    var field = SampleEntity.class.getDeclaredField("someValue");
+    assertEquals("some_value", EntityUtil.resolveFieldColumnName(field));
+  }
+
+  @Test
+  void resolveFieldColumnValueWithAnnotation() throws NoSuchFieldException {
+    var field = AnnotatedSampleEntity.class.getDeclaredField("someValue");
+    assertEquals("custom_column_name", EntityUtil.resolveFieldColumnName(field));
+  }
+
+  @Test
+  void resolveEntityIdField() throws NoSuchFieldException {
+    var expectedIdField = SampleEntity.class.getDeclaredField("id");
+    assertEquals(expectedIdField, EntityUtil.resolveEntityIdField(SampleEntity.class));
+  }
+
+  @Test
+  void resolveEntityIdFieldShouldThrowMalformedEntityExceptionWhenNoIdFound() {
+    assertThrows(MalformedEntityException.class, () -> EntityUtil.resolveEntityIdField(EntityWithoutId.class));
+  }
+
+  @Test
+  void resolveEntityIdFieldShouldThrowMalformedEntityExceptionWhenMultipleIdFound() {
+    assertThrows(MalformedEntityException.class, () -> EntityUtil.resolveEntityIdField(EntityWithMultipleIds.class));
+  }
```

Comment:
In these two cases MalformedEntityException is thrown, so it would be more precise also to check an error message